### PR TITLE
Update docstring function

### PR DESCRIPTION
**Pull request by nabobalis**
Trying to fix the docs.

Changed from shared to a comment so avoid it showing up in the rst pages.

### CODE REVIEW

File: hvpy/helpers.py [hunk 1]
```diff
@@ -65,6 +64,7 @@ def createMovie(
     timeout
         The timeout in minutes to wait for the movie to be created.
         Default is 5 minutes.
+    {Insert}
```
Comment (nabobalis) [inline hvpy/helpers.py:67]: This is one way to do it, even tho it is technically in the wrong order.

**Comment by akash5100** [comment]
In the createMovie function:
```
def createMovie(
    startTime: datetime,
    endTime: datetime,
    layers: str,
    events: str,
    eventsLabels: bool,
    imageScale: float,
    format: Optional[str] = "mp4",
    frameRate: Optional[str] = "15",
    maxFrames: Optional[str] = None,
    scale: Optional[bool] = None,
    scaleType: Optional[str] = None,
    scaleX: Optional[float] = None,
    scaleY: Optional[float] = None,
    movieLength: Optional[float] = None,
    watermark: Optional[bool] = True,
    width: Optional[str] = None,
    height: Optional[str] = None,
    x0: Optional[str] = None,
    y0: Optional[str] = None,
    x1: Optional[str] = None,
    y1: Optional[str] = None,
    x2: Optional[str] = None,
    y2: Optional[str] = None,
    size: Optional[int] = None,
    movieIcons: Optional[int] = None,
    followViewport: Optional[int] = None,
    reqObservationDate: Optional[datetime] = None,
    overwrite: bool = False,
    filename: Union[str, Path] = None,
    hq: bool = False,
    timeout: float = 5,
) -> Path:
```
The `callback` argument(which is in queueMovie) returns the string response. We only want to write binary data in the `createMovie` function. So to eliminate the option of returning str, I removed the `callback` argument. I did right?

**Comment by nabobalis** [comment]
Probably.

**Comment by akash5100** [comment]
> Probably.

So if we copy docstring, it will copy `callback` too. That is the problem.

**Comment by nabobalis** [comment]
Yes that is a problem.

**Pull request merged**